lib/ansible/plugins/connection/vmware_tools.py

```diff
@@ -0,0 +1,309 @@
+"""Docstring for public module."""
```

Comment:
Shouldn't this contain some license/copyright info and info on the plugin? (never developed a connection plugin, but it seems like a good thing to have)

Reply:
added the standard (I hope) copyright / license